build.gradle

```diff
@@ -429,6 +429,8 @@ project(':iceberg-spark') {
     compile project(':iceberg-parquet')
     compile project(':iceberg-hive')
 
+    compile 'de.javakaffee:kryo-serializers'
```

Comment:
I think this should be included in iceberg-spark-runtime, which means that we will need to make sure we add it to the shaded LICENSE and NOTICE.

Reply:
Added additional LICENSE and NOTICE.